src/assertions/text.js

```diff
@@ -0,0 +1,12 @@
+
+import should from 'should';
+
+const Assertion = should.Assertion;
+Assertion.add('text', function(string) {
+  const text = this.obj.text();
+  this.params = {
+    obj: this.obj.name(),
+    operator: `to have text '${string}' but found '${text}'`,
+  };
+  should(text.includes(string)).be.true(' ');
```

Comment:
The issue i had is that bool assertioon builder is built to take args and pass them into an enzyme method (className, props, etc). ContainsText you take that argument call a text method with no arguments and compare the results with the original string.